packages/node-cli/src/utils/AccountCredentialsFormatter.js

```diff
@@ -42,22 +42,21 @@ function _getAccountType(accountId) {
 	return NodeTranslationService.getMessage(ACCOUNT_TYPE.PRODUCTION);
 }
 
-function getListItemString(authID, accountCredential) {
+function getListItemString(authID, accountCredentials) {
 	const isNotProductionUrl =
-		accountCredential.urls &&
-		!accountCredential.urls.app.match(DOMAIN.PRODUCTION.PRODUCTION_DOMAIN_REGEX) &&
-		!accountCredential.urls.app.match(DOMAIN.PRODUCTION.PRODUCTION_ACCOUNT_SPECIFIC_DOMAIN_REGEX);
+		accountCredentials.urls &&
+		!accountCredentials.urls.app.match(ApplicationConstants.DOMAIN.PRODUCTION.PRODUCTION_DOMAIN_REGEX) &&
```

Comment:
I think you can delete the previous import DOMAIN related to AccountCredentials (line10)

Reply:
I can't. I am using it at line 30. That was the fix :)